arrow/parser.py

```diff
@@ -429,23 +441,23 @@ def _choice_re(choices, flags=0):
 
 
 class TzinfoParser(object):
-
-    _TZINFO_RE = re.compile(r"([+\-])?(\d\d):?(\d\d)?")
+    # TODO: test against full timezone DB
+    _TZINFO_RE = re.compile(r"^([\+\-])?(\d{2})(?:\:?(\d{2}))?$")
```

Comment:
again maybe we should go through this on zoom

Reply:
Yup! We can talk through this, but here is another appetizer: https://regex101.com/r/4NnaLU/1.